packages/web-components/src/components/cbp-structured-list-item/cbp-structured-list-item.tsx

```diff
@@ -28,6 +28,12 @@ export class CbpStructuredListItem {
     });
   }
 
+  @Listen('stateChanged')
+  stateChangedHandler(){
+    console.log('state toggle for structured list item selected');
+    this.selected = !this.selected;
```

Comment:
You should be keying the selected value off of the checked value emitted in that event. Let me know if you want to talk through this. I think pagination is a good example where I'm using destructuring of custom events to get at a specific key.

Reply:
update to use map the selected prop to the checked value of provided by the event. Used the   @Listen('buttonClick') listener from the pagination as an example for the deconstructing of a custom event